src/main/com/aca/homework/week7/singleton/SingletonUserService.java

```diff
@@ -0,0 +1,23 @@
+package com.aca.homework.week7.singleton;
+
+public class SingletonUserService {
+    private static SingletonUserService user;
+
+    private SingletonUserService(){}
+
+    public static SingletonUserService getInstance(){
+        if(userExists(user)){
```

Comment:
for this case, we do not need to create userExists and assignUserToNewObject methods. the code is very simple and there is no need to create those methods